src/mono/wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj

```diff
@@ -33,6 +33,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Playwright" Version="1.21.0" />
+    <ProjectReference Include="..\..\..\tasks\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks.csproj" />
```

Comment:
We shouldn't be adding direct project dependencies like this. This should be getting built, and packages generated as part of rest of the build. Inner loop improvements can be made in other ways.

Reply:
The reason for the reference is to be able to use `BootJsonData` class.
This is how are tests composed in SDK. Why shouldn't we use it here? What solution do you propose?

Reply:
This is fine for now.